GoogleCloudPlatform/datacatalog-connectors-bi
# Set the order of the Tag Template fields

**- What I did**
Set the order of the Tag Template fields, so their related Tag fields will always be displayed in the same order in the Data Catalog UI.

**- How I did it**
By specifying a value for the `order` attribute of all Tag Template fields.

**- How to verify it**
Run the connector in an integrated environment. Then, verify the order of the Tags fields to make sure they are presented in the same order they were added to their respective Tag Templates.

**- Description for the changelog**
Set the order of the Tag Template fields
